.github/workflows/test_dispatch.yml

```
@@ -0,0 +1,19 @@
+name: "Dispatch"
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Version tag"
+        required: true
+        type: string
+
+jobs:
+  test:
+    name: Test
+    uses: EventStore/EventStore-Client-NodeJS/.github/workflows/tests.yml@master
```

```suggestion
    uses: ./.github/workflows/tests.yml
```

would it be preferable to [reference the workflow locally](https://github.blog/changelog/2022-01-25-github-actions-reusable-workflows-can-be-referenced-locally/) ?

---

I assume it's a relative path to the root of the repo not relative to this workflow file? Maybe we'd be better off leaving it matching the other workflow files, then updating them all once we know this one works.

---

ok, sounds good :+1: